addon-update-action.py

```diff
@@ -29,8 +29,18 @@
 
     addon_repo = github.get_repo(f"{ORGANIZATION}/{ADDON_REPO}")
 
+    pre_release_bool = False
+
+    addon_version = "music_assistant"
+
+    if "b" in args.new_server_version:
+        pre_release_bool = True
+
+    if pre_release_bool is True:
+        addon_version = "music_assistant_beta"
```

Comment:
I think you need to pass this from the release script because that is already figuring our if its prerelease from github itself ?

Reply:
True, I suppose I can just extract that from somewhere.